### PR TITLE
Migrate navdrawer and scroller to ES6

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**

Cleanup here was very minimal since:
* Both libraries are supposed to be external (but not available on npm, and modified)
* They're abandoned
* They will be replaced with Vue components soon.

Make sure to disable whitespace changes when reviewing.

Part of #868
Needs #1675 for the linting fixes to screensavermanager.

**Issues**

### CODE REVIEW

File: src/libraries/scroller.js [hunk 1]
```diff
@@ -1,928 +1,886 @@
...
-                    container.scrollTop = Math.round(pos);
-                }
+                container.scroll({
+                    top: pos,
+                    behavior: immediate ? 'instant' : 'smooth'
+                });
+            }
+        } else if (!immediate && container.scrollTo) {
+            if (o.horizontal) {
+                container.scrollTo(Math.Math.round(pos), 0);
```
Comment (Camc314) [inline src/libraries/scroller.js:232]:
```suggestion
                container.scrollTo(Math.round(pos), 0);
```
Math.Math....?

File: src/libraries/scroller.js [hunk 2]
```diff
@@ -1,928 +1,886 @@
...
+            } else {
+                container.scrollTo(0, Math.Math.round(pos));
```
Comment (Camc314) [inline src/libraries/scroller.js:234]:
```suggestion
                container.scrollTo(0, Math.round(pos));
```

File: src/libraries/scroller.js [hunk 3]
```diff
@@ -1,928 +1,886 @@
...
+            }
+        } else {
+            if (o.horizontal) {
+                container.scrollLeft = Math.Math.round(pos);
```
Comment (Camc314) [inline src/libraries/scroller.js:238]:
```suggestion
                container.scrollLeft = Math.round(pos);
```

File: src/libraries/scroller.js [hunk 4]
```diff
@@ -1,928 +1,886 @@
...
+            } else {
+                container.scrollTop = Math.Math.round(pos);
```
Comment (Camc314) [inline src/libraries/scroller.js:240]:
```suggestion
                container.scrollTop = Math.Math.round(pos);
```

File: src/libraries/scroller.js (hunk 1 above)
Comment (MrTimscampi) [reply src/libraries/scroller.js:232]: Search and replace failed me again :D

File: src/libraries/scroller.js [hunk 5]
```diff
@@ -1,928 +1,886 @@
...
-
-    browser = browser.default || browser;
-    focusManager = focusManager.default || focusManager;
-
-    /**
+/* Cleaning this file properly is not neecessary, since it's an outdated library
+ * and will be replaced soon by a Vue component.
+ */
+
+ import browser from 'browser';
```
Comment (dmitrylyzo) [inline src/libraries/scroller.js:5]: lint
```suggestion
import browser from 'browser';
```

**Pull request merged**